kiwi_lint/similar_string.py

```diff
@@ -0,0 +1,73 @@
+import string
+
+import astroid
+from pylint.checkers import BaseChecker, utils
+from pylint.interfaces import IAstroidChecker
+from textdistance import levenshtein
+
+
+class SimilarStringChecker(BaseChecker):
+    __implements__ = (IAstroidChecker,)
+    name = "similar-string-checker"
+
+    msgs = {
+        "R5011": (
+            'A similar string to "%s" exists: "%s"',
+            "similar-string",
+            "Similar strings should be avoided for single translation",
+        )
+    }
+
+    def __init__(self, linter):
+        super().__init__(linter)
+        self._dict_of_strings = {}
+
+    @staticmethod
+    def clean_string(text):
+        for word in text:
+            if word not in string.punctuation:
+                text += "".join(word)
+        return text
+
+    @staticmethod
+    def marked_for_translation(funcname):
+        if funcname in ("_", "gettext_lazy"):
```

Comment:
the if condition is a boolean expression in itself so no need for the if-return construct.